cardano-testnet/src/Testnet/TestQueryCmds.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE GADTs #-}
+{-# OPTIONS_GHC -Wno-missing-signatures #-}
+{-# OPTIONS_GHC -Wno-unused-top-binds #-}
+
+module Testnet.TestQueryCmds
+  ( TestQueryCmds(..)
+  , forallQueryCommands
+  ) where
+
+import Cardano.CLI.EraBased.Commands.Query (QueryCmds (..))
+
+import Testnet.TestEnumGenerator (genTestType, genAllConstructorsList)
+
+$(genTestType ''QueryCmds)
```

Comment:
Can you add a comment that explicitly states what the generated type declarations is i.e `TestQueryCmds`

Reply:
Solved [here](https://github.com/IntersectMBO/cardano-node/commit/786e534b8020e928aeaf35e3ea24d57fab6615f8)